src/server/routes/api.routes.js

```diff
@@ -21,14 +21,16 @@ ApiRoutes.post('/uploadimage', ensureAuthenticated, upload.single('image'), asyn
 
     const uploadRes = (await req.callServiceProvider('uploadimage', {image: req.file, accessToken}))[0];
     if (uploadRes.statusCode >= 400) {
+      log.error('Error occurred during file upload!');
```

Comment:
Jeg er lidt i tvivl om det er i scope for denne opgave, og om det er et problem, men her logger vi en tekststring som en fejl. På linje 33 logges et object err og på linje 57 logges en besked og et fejlobjekt. 

I øvrigt bliver der kastet en fejl på næste linje, så denne fejl bliver logget to gange.